h/static/scripts/base/sentry.ts

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
This has the same structure as the old `raven.js` module, but converted to TS and adapted to use the new Sentry API. I also removed the unhandled-promise code as that is obsolete.